src/App.js

```diff
@@ -14,11 +14,53 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selected: 'Jam',
+  };
+
+  render() {
+    const { selected } = this.state;
+
+    return (
+      <div className="App">
+        {selected ? (
+          <h1>
+            {`${selected} is selected`}
+            <button
+              className="cancel"
+              type="button"
+              onClick={() => {
+                this.setState({ selected: null });
+              }}
+            >
```

Comment:
is better to create a method and use it as a callback

Reply:
Ok, this one is short so I put it inside, but yeah - callbacks looks better